.github/workflows/generate-package-metadata.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - id: set-matrix
-        run: echo "::set-output name=matrix::$(python generate_package_list.py)'"
+        run: echo "matrix=$(python generate_package_list.py)'" >> $GITHUB_OUTPUT
         working-directory: community-packages
       - name: List open pull requests
         uses: jkisk/list-open-pulls@v1.0.1
```

Comment:
This is a custom action that is still using set-output. We will continue to get the deprecation errors until this is updated. This code is managed in jamie's personal repo. I think I will likely need to just do a fork of this to fix it or find another way to list out the pull requests maybe using a shell script or something.

https://github.com/jkisk/list-open-pulls/blob/main/index.js#L19